wercker.yml

```diff
@@ -1,9 +1,63 @@
-box: wercker-labs/docker
+box: centos:6.7
```

Comment:
In new Wercker build stack, this field should be used for specifying DockerHub's image name.